build.gradle.kts

```diff
@@ -13,12 +13,13 @@ version = "2.2.2-SNAPSHOT"
 
 repositories {
     jcenter()
+    google()
 }
 
 dependencies {
     compileOnly(gradleApi())
     compileOnly(kotlin("gradle-plugin", "1.1.1"))
-    compileOnly("com.android.tools.build:gradle:2.3.3")
+    compileOnly("com.android.tools.build:gradle:3.0.0-rc2")
```

Comment:
`3.0.0` is already available, please update to it.

Reply:
sure